elf/elf.go

```diff
@@ -566,6 +566,20 @@ func (b *Module) Load(parameters map[string]SectionParams) error {
 				progType = uint32(C.BPF_PROG_TYPE_SCHED_ACT)
 			}
 
+			// If Kprobe or Kretprobe for a syscall, use correct syscall prefix in section name
+			if isKprobe || isKretprobe {
+				str := strings.Split(secName, "/")
+				if (strings.HasPrefix(str[1], "SyS_")) || (strings.HasPrefix(str[1], "sys_")) {
+					name := strings.TrimPrefix(str[1], "SyS_")
+					name = strings.TrimPrefix(name, "sys_")
+					syscallFnName, err := GetSyscallFnName(name)
+					if err != nil {
+						return err
```

Comment:
Shouldn't we just ignore the error in that case? If `/proc/kallsyms` is not readable, we could just keep the name from the ELF file.

Reply:
Yes, makes sense